### PR TITLE
Add @babel/polyfill to npm dependencies

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix

## Description
This adds `@babel/polyfill` to npm dependencies in order to support IE11 and some Edge versions.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const config = {
   mode: isProduction ? "production" : "development",
   entry: {
     app: [
+      "@babel/polyfill",
```
Comment (gabrieldutra) [inline webpack.config.js]: Not sure if this is the best place to put this (could be in `index.js` as well)

Comment (kravets-levko) [reply webpack.config.js]: I think it's a good place for such things as well.

**Comment by ranbena** [comment]
So I think we're unanimous that Redash should no longer support IE11.
Perhaps polyfill promise only (https://stackoverflow.com/a/41062991/259289)

**Comment by arikfr** [comment]
I opened #3579 about showing a message to IE users.
Closing this now.

**Pull request closed without merging**